pkg/morph/client/container/meta.go

```diff
@@ -0,0 +1,27 @@
+package container
+
+import (
+	"fmt"
+
+	"github.com/nspcc-dev/neofs-node/pkg/morph/client"
+)
+
+// SubmitObjectPut puts object meta information.
+//
+// Returns any error encountered that caused the saving to interrupt.
+func (c *Client) SubmitObjectPut(meta []byte, sigs [][]byte) error {
+	if len(meta) == 0 || len(sigs) == 0 {
```

Comment:
so `sigs` required? https://github.com/nspcc-dev/neofs-contract/pull/448#discussion_r1864201887

Reply:
your link is a unit test for contract API. it was tested with not-a-real container with no real REP values, so it did not require any signatures (kinda ok: you said you wanted no REP, the contract proved that at least 0 REP is validated)

on the other hand, this is a client side that wants to prove that somebody stores an object (non-zero REP, otherwise -- why?). it is senseless to provide no signatures. if REP > 0 and you send no signatures, it will fail